### PR TITLE
Logcat Pinging improvements

### PR DESCRIPTION
**Pull request by ujappelbe**
- Better output
- Less trash output
- Source-fix so can be used from start-script

### CODE REVIEW

File: logcat_pinger.sh [hunk 1]
```diff
@@ -7,14 +7,16 @@ export TEST_RUNNING_FILE="$(mktemp)"
...
-  PINGS_IN_BLOCK=10
+  PINGS_IN_BLOCK=4
+  INTERVAL=3
   touch "${TEST_RUNNING_FILE:?}"
   (
     while [ -f "${TEST_RUNNING_FILE:?}" ]
     do
       date_at_start=$(date +%s)
-      adb shell "ping -c${PINGS_IN_BLOCK} $1 |while read line ; do log -pd -tQApinger \$line ; done"
+      adb shell "ping -c${PINGS_IN_BLOCK} -i${INTERVAL} $1 |while read line  ; do line=\$(echo \$line | sed -e 's/-/_/g') ; log -pd -tQApinger \"\$line .\" ; done"
```
Comment (shusso) [inline logcat_pinger.sh:19]: this is starting to get quite long. Should it be an function of it's own?

File: logcat_pinger.sh [hunk 2]
```diff
@@ -7,14 +7,16 @@ export TEST_RUNNING_FILE="$(mktemp)"
 # Args:
 # $1 host_to_ping
 function ping_to_logcat {
+  echo "ping_to_logcat TEST_RUNNING_FILE=$TEST_RUNNING_FILE"
   MINIMUM_LOOP_TIME=5
-  PINGS_IN_BLOCK=10
+  PINGS_IN_BLOCK=4
```
Comment (antnsi) [inline logcat_pinger.sh:12]: have no idea what is this, plz add comments for future

File: logcat_pinger.sh (hunk 1 above)
Comment (antnsi) [inline logcat_pinger.sh:19]: agree, no oneliners plz.

File: test_start_helpers.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # Helpers for commandline scripts related to appium tests.
-
-source "logcat_pinger.sh"
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```
Comment (antnsi) [inline test_start_helpers.sh:3]: i would add if [ ! -d "${BASH_SOURCE[0]}" ]; then error; fi before this call

File: logcat_pinger.sh (hunk 1 above)
Comment (ujappelbe) [inline logcat_pinger.sh:19]: it must be oneline currently as it is run on the device, not on the host computer. 

But actually I was planning of making it into a script that is uploaded to the device, if that works.

File: logcat_pinger.sh (hunk 2 above)
Comment (ujappelbe) [inline logcat_pinger.sh:12]: That is true, I'll add comments to this script.

File: test_start_helpers.sh (hunk 3 above)
Comment (ujappelbe) [inline test_start_helpers.sh:3]: For a case where a different shell than bash is used? I can add the check.

**Comment by ujappelbe** [comment]
Since this did not actually work on testdroid I will close this PR, create a separate script that is uploaded to device and hope that works in TD environment.

**Pull request closed without merging**